trulens_eval/examples/snowflake_demo/feedback.py

```diff
@@ -99,12 +99,16 @@ def get_feedbacks(provider_name: str, use_rag: bool = True):
 
     if use_rag:
         return [
-            f_context_relevance,
-            f_small_local_models_context_relevance,
-            f_answer_relevance,
-            f_groundedness,
-            f_criminality_input,
-            f_criminality_output,
-        ]
+          f_context_relevance,
+          #f_small_local_models_context_relevance,
```

Comment:
won't it be easier to see the context relevance guard rail decisions if this is in?

Reply:
AD/SS view is that it clutters the dashboard to show this/could be confusing to the audience